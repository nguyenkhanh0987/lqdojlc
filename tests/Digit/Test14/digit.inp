100
554253879 861205596
443002358 532767281
578028095 607182783
913927657 965408154
551068555 860838948
455820278 802685044
302000898 572128453
688496752 821989172
340527183 465466933
587051419 928990674
244502814 296541032
298777661 317553405
501325064 759047090
965856858 978571222
481645524 992769092
634878467 903274031
632699872 886979955
943981252 992239973
522245549 909327434
772960680 836164082
440104479 763938069
787882437 838109493
310901912 756838126
562819537 878856236
71673681 77788790
77505634 324294003
583232201 995485222
580778568 812725490
594397322 727528823
83032294 282715989
796072767 995106170
89409945 515740091
468954185 622090479
884694805 889550655
565227578 842522038
365924041 543123880
578947961 718649888
42825782 100177789
820681373 975678277
709778178 800046227
172680948 926561735
56663001 82735172
980941955 994457811
898098991 970077543
724131855 842080279
4183104 16402123
119962915 394080182
225527738 971444610
503756690 815951263
521406396 816714623
105273045 809801501
800137601 942907280
244027749 990981783
140141369 436458468
886832268 896399249
87129606 282057336
622893279 779231872
264040629 867163105
611460275 871359337
328366891 652091385
938308886 961843837
1018101 767803112
50224445 153071987
955485001 967268627
142039089 391630206
688862305 929051502
223492145 857052145
514859607 901923699
625652701 858001476
886483544 886810877
436832086 787890875
465796020 592005228
274284772 584038344
569104449 916898874
165657960 595286088
307131574 482552465
321543374 500955383
549401358 992226593
4390926 113784792
606605034 607404427
671442593 733378912
495062364 814289029
285373499 319855615
40110719 826259272
573613268 844947250
509702511 536615760
285528586 980633835
640294854 996368017
971488610 978057853
511877258 965944760
13832906 379565442
224704933 589983112
441617802 503715380
23452183 604651342
346251725 561962543
333311217 878167084
570238835 735460714
210440004 744641643
492030582 661776359
770394392 907431208
