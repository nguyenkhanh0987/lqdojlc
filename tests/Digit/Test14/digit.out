29117652
11982977
4362944
2338445
29585599
38927426
35685973
11297014
18485259
29140711
11056090
1749382
25260756
969772
47087059
24264910
22757687
3833794
36795462
4989127
36392145
3101120
53478281
29734604
1637878
44953210
34833187
22894410
12209317
36997796
13277094
69587683
19114967
402001
26405794
23995137
14789071
14199440
10222818
8381348
91985154
6446249
1636241
2967826
9264625
1607232
48475958
84600527
30903115
29966915
95776508
8547828
83859464
49667386
1317700
35671583
14627837
71462174
22154153
41767308
1327262
112776002
20856904
781080
44174202
18504143
77156568
37152352
20178632
36473
39488266
16939082
44138401
32028531
66055116
24845920
27298517
39490398
22029864
15586
6036042
32011484
5105395
112076664
25349905
2066460
74339717
29677232
604874
40008607
66294368
53489832
10353989
95356259
29471300
61062307
17239240
69745710
17934111
11631455
